SpatialGDK/Source/SpatialGDK/Private/Utils/OpUtils.cpp

```diff
@@ -22,6 +22,22 @@ void FindFirstOpOfType(const TArray<Worker_OpList*>& InOpLists, const Worker_OpT
 	}
 }
 
+void FindAllOpsOfType(const TArray<Worker_OpList*>& InOpLists, const Worker_OpType InOpType, TArray<Worker_Op*>& FoundOpArray)
```

Comment:
I'd possibly change the name to AppendAllOpsOfType. Otherwise I'd expect it to return an array or soemthing.

Reply:
I think I would prefer the OutFoundOps, to keep consistency with the other functions here, such as FindFirstOpOfType.

Reply:
Discussed offline - the other function are using out params. This is an in-out param. And the other functions should really be using return values (which they are in the spatialview pr).
Here we can't use a return value and we also expect to preserve existing state in the array passed in.

Overall, for the clearest function I'd rename it to AppendAllOpsOfType and not prepend anything to the array argument name.